Pheno/scripts/histogram_script.R

```diff
@@ -0,0 +1,42 @@
+library(ggplot2)
+
+setwd("/Users/dana.alhindi/DDRIG/Pheno/data_private")
+
+# Plotting histograms for SA pigmentation data from 4 populations using ggplot2
```

Comment:
If you want this comment I would put it at the top of the file before the libraries. Usually we don't, but this is kind of setting up context for understanding so it's ok. Also something more descriptive would be helpful. You aren't just plotting pigmentation data, you're plotting x or y or z types of genes chromosomes etc for blah blah visualization and etc etc analysis. You know?